woocommerce/woocommerce
# Feature integration: UI Revamp on Marketing Page

### All Submissions:


### Changes proposed in this Pull Request:

This PR contains the whole functionality for the epic issue https://github.com/woocommerce/woocommerce/issues/33895. It merges the main feature branch into the trunk branch. It consists of the following subfeature PRs that have been reviewed and merged:

- https://github.com/woocommerce/woocommerce/pull/33964
- https://github.com/woocommerce/woocommerce/pull/34080
- https://github.com/woocommerce/woocommerce/pull/34264
- https://github.com/woocommerce/woocommerce/pull/34415
    - https://github.com/woocommerce/woocommerce/pull/34444

I am hoping to get this merged and make it into WooCommerce 7.0 release, if that is possible. (Code freeze for WC 7.0 is on Thursday 15 September 2022).

I will come up with more tests (playwright e2e tests and unit tests) in separate PR later. The test PR would be targeting trunk branch directly after this PR.

### How to test the changes in this Pull Request:

The following is a brief test. For more detailed tests instructions with screenshots and demo videos, please visit the subfeature PRs above.

1. Go to WooCommerce Settings > Advanced > Features, and enable the multichannel marketing feature.
2. Go to the Marketing page at `/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`. You should see the new beta Multichannel Marketing page UI.
3. In the page, you should see the "Discover more marketing tools" card. It should be collapsed by default. Expand the card and you should see a list of marketing extensions. 
4. You should see the "Learn about marketing a store" card. It should be collapsed by default.
5. Install a marketing extension, and come back to the Marketing page. 
6. You should see the "Installed extensions" card. The card should be expanded by default. 
7. Go back to WooCommerce Settings > Advanced > Features, and disable the multichannel marketing feature.
8. Go to the Marketing page at `/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`. You should see the existing normal Marketing page UI.

### Other information:


### FOR PR REVIEWER ONLY:
